content/applications/hr/employees/new_employee.rst

```diff
@@ -79,188 +62,269 @@ Resumé tab
 section.
 
-Work information tab
+.. _employees/skill-types:
+
+Skill Types
+***********
+
+In order to add a skill to an employee's form, the :guilabel:`Skill Types` must be configured. Go to
+:menuselection:`Employees app --> Configuration --> Skill Types` to view the currently configured
+skill types and create new skill types. Click :guilabel:`New` and a new :guilabel:`Skill Type`
+form appears. Fill out all the details for the new skill type. Repeat this for all the skill types
+needed.
+
+- :guilabel:`Skill Type`: Enter the name of the skill type. This should be somewhat generic, since
+  the specific skills listed will be housed under this category.
+- :guilabel:`Skills`: Click :guilabel:`Add a line` and enter the name for the new skill, then repeat
+  for all other needed skills.
+- :guilabel:`Levels`:  Click :guilabel:`Add a line` and a :guilabel:`Create Levels` form appears.
+  Enter the name of the level, and assign a :guilabel:`Progress` percentage (0-100) for that level.
+  Click :guilabel:`Save & New` to save the entry and add another level, or click :guilabel:`Save &
+  Close` to save the level and close the form.
+
+  .. example::
+     To add a math skill set, enter `Math` in the :guilabel:`Name` field. Next, in the
+     :guilabel:`Skills` field, enter `Algebra`, `Calculus`, and `Trigonometry`. Last, in the
+     :guilabel:`Levels` field enter `Beginner`, `Intermediate`, and `Expert`, with the
+     :guilabel:`Progress` listed as `25`, `50`, and `100`, respectively. Then, either click
+     :guilabel:`Save & Close` or :guilabel:`Save & New`.
+
+       .. image:: new_employee/math-skills.png
+          :align: center
+          :alt: Add new math skills and levels with the skill types form.
+
+Work Information tab
 --------------------
 
-- :guilabel:`Location`: Select the :guilabel:`Work Address` from the drop-down menu. The
-  :guilabel:`External Link` button opens up the selected company form in a window, and allows for
-  editing. The :guilabel:`Work Location` is where any specific location details should be noted,
-  such as a floor, or building.
+The Work Information tab is where the employee's specific job related information is housed. Their
+working schedule, various roles, who approves their specific requests (time off, timesheets, and
+expenses), and specific work location details are listed here. Enter the following information for
+the new employee.
+
+- :guilabel:`Location`: Select the :guilabel:`Work Address` and :guilabel:`Work Location` from the
+  corresponding drop-down menus. The work address :guilabel:`Internal Link` button opens up the
+  selected company form in a window, and allows for editing. The :guilabel:`Work Location` is the
+  specific location details, such as a floor or building. If a new work location is needed, add the
+  location by typing it in the field.
 - :guilabel:`Approvers`: Using the drop-down menus, select the employees responsible for approving
```

Comment:
```suggestion
- :guilabel:`Approvers`: Using the drop-down menus, select the users responsible for approving
```
same as with 15, this is a small change, but since you can only choose a user here, not just an employee, I thought it made sense to make that distinction.